CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 
-All notable changes to this project will be documented in this file. The format is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project follows the versions of [SMASH](https://github.com/smash-transport/smash), that means that each new version of SMASH triggers a new version of this project, in order to assure compatibility.
+All notable changes to this project will be documented in this file. The format is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project follows the versions of [SMASH](https://github.com/smash-transport/smash), that means that each new version of SMASH triggers a new version of this project, in order to assure compatibility. A [release checklist](#release-checklist) is provided at the end of the file. The described procedure should be followed for every new release.
```

Comment:
I would:
* use one sentence per line for easier review 😉 
* specify that therefore new intermediate releases of the handler itself will use the third digit of the versioning number, which is then not necessarily reserved to bug-fixes.